src/core/buffers/representation/get_needed_segments.ts

```diff
@@ -61,10 +66,20 @@ export default function getNeededSegments({
   content,
   currentPlaybackTime,
   knownStableBitrate,
-  loadedSegmentPendingPush,
   neededRange,
-  segmentInventory,
+  queuedSourceBuffer,
 } : ISegmentFilterArgument) : ISegment[] {
+  /** Inventory of every segments currently in the SourceBuffer. */
```

Comment:
Are you sure you want to comment here ? It seems a little bit unecessary, no ?

Reply:
Ok